iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ClientConfiguration.java

```diff
@@ -86,6 +86,10 @@
     @Setter(AccessLevel.PACKAGE)
     private int sendInterval = DEFAULT_SEND_INTERVAL_IN_MILLISECONDS;
 
+    @Getter
+    @Setter(AccessLevel.PACKAGE)
+    private boolean isConnectingToMqttGateway; //TODO populate this field, Bryce
```

Comment:
FYI, this is addressed in [here](https://github.com/Azure/azure-iot-sdk-java/pull/1708/commits/36dd289dfcd68712a0d6ae14d8fa9cea4620351e)

Reply:
my PR merged in